config.json

```diff
@@ -259,7 +259,11 @@
     {
       "difficulty": 1,
       "slug": "luhn",
-      "topics": []
+      "topics": [
+	"Algorithms",
+	"Integers",
+	"Strings"
```

Comment:
[My solution](http://exercism.io/submissions/3638377f9a8c4e869b0c2a0c724ff02f). It could be a 2, if we look at 1 as the most basic "hello world" stuff only. I'm not sure what else would qualify as a 1 in the track though.